juno/assetlist.json

```diff
@@ -2077,6 +2077,51 @@
         }
       ],
       "coingecko_id": "backbone-labs-staked-juno"
+    },
+    {
+      "type_asset": "ics20",
+      "description": "USD Coin on Juno",
+      "denom_units": [
+        {
+          "denom":
+            "ibc/4A482FA914A4B9B05801ED81C33713899F322B24F76A06F4B8FE872485EA22FF",
+            "exponent": 0,
+            "aliases": ["uusdc", "microusdc"]
+        },
+        {
+          "denom": "usdc",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/4A482FA914A4B9B05801ED81C33713899F322B24F76A06F4B8FE872485EA22FF",
+      "name": "USD Coin",
+      "display": "usdc",
+      "symbol": "USDC",
+      "coingecko_id": "usd-coin",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/noble/images/USDCoin.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/noble/images/USDCoin.svg"
+      },
+      "images": [
+        {
```

Comment:
can you please include image sync?
"image_sync": {
  "chain_name": "noble",
  "base_denom": "uusdc"
}